### DISCUSSION TITLE
curl vulnerability

### DISCUSSION THREAD

**Discussion by dtrace4linux** [curl/curl#14395] [General]
I have an issue with curl - a specific idea/feature/improvement, to address a vulnerability in curl. This is not some classical "do-this and hey! you p0wned" a system, but something that curl is ignoring. The issue isnt specific to curl, but thought it worth raising, because curl is so very awesome, yet this feature is missing.

I wont highlight the issue in this public forum.

**Comment by bagder** [comment]
For "non-disclosed" discussions and security reporting to the curl project, use hackerone: https://hackerone.com/curl

**Reply by bagder** [reply]
Optionally, email the curl security team at security@curl.se

**Comment by dtrace4linux** [comment]
Super - thanks Daniel - will do.
